packages/shell-chrome/assets/panel.html

```diff
@@ -20,84 +20,127 @@
             class="h-full"
             x-cloak
             x-data="{
```

Comment:
I think it's probably time to move the `x-data`/`x-init` into a function in a JS file. It would allow us to stop sticking stuff onto globals in `init()` (app.js file)

Doesn't need to happen in this PR, we can do it separately

Reply:
Yeah I almost broached that in this PR. It'd probably be a good way to go at this point.